stroom-pipeline/src/test/java/stroom/pipeline/xsltfunctions/TestHexToString.java

```diff
@@ -0,0 +1,81 @@
+package stroom.pipeline.xsltfunctions;
+
+import stroom.util.logging.LambdaLogger;
+import stroom.util.logging.LambdaLoggerFactory;
+import stroom.util.shared.Severity;
+
+import net.sf.saxon.om.EmptyAtomicSequence;
+import net.sf.saxon.om.Sequence;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TestHexToString extends AbstractXsltFunctionTest<HexToString> {
+
+    private static final LambdaLogger LOGGER = LambdaLoggerFactory.getLogger(TestHexToString.class);
+
+    private HexToString hexToString;
+
+    @BeforeEach
+    void setUp() {
+        hexToString = new HexToString();
+    }
+
+    @Test
+    void call() {
```

Comment:
Can you split these up into multiple tests.  If we get a problem, having >1 case in a test makes it harder to debug a single case.  Either create multiple tests, use `@TestFactory` or use `TestUtil.buildDynamicTestStream` (as used in `TestHash`).